site/themes/docsy/layouts/partials/head.html

```diff
@@ -34,7 +33,7 @@
 {{end}}
 {{ if .Site.Params.prism_syntax_highlighting }}
 <!-- stylesheet for Prism -->
-<link rel="stylesheet" href="{{ "/css/prism.css" | relURL }}"/>
+<link rel="stylesheet" href="{{ "/site/prism.css" | absURL }}"/>
```

Comment:
```suggestion
<script src='{{ "js/prism.js" | relURL }}'></script>
```

Reply:
Updated it. `hugo-test` failed with two errors. Details: https://gist.github.com/Kalaiselvi84/7f12221fa15b8e8707587039fcba59ec